Tools/TestWebKitAPI/Tests/WTF/Expected.cpp

```diff
@@ -161,12 +161,24 @@ TEST(WTF_Expected, expected)
         EXPECT_EQ(u.error(), oops);
         EXPECT_EQ(u.value_or(3.14), 3);
     }
+    {
+        auto u = E(unexpect, oops);
+        EXPECT_FALSE(u.has_value());
+        EXPECT_EQ(u.error(), oops);
+        EXPECT_EQ(u.value_or(3.14), 3);
+    }
     {
         auto uv = EV(makeUnexpected(oops));
         EXPECT_FALSE(uv.has_value());
         EXPECT_EQ(uv.error(), oops);
         EXPECT_EQ(uv.value_or(3.14), 3);
     }
+    {
+        auto uv = EV(unexpect, oops);
+        EXPECT_FALSE(uv.has_value());
+        EXPECT_EQ(uv.error(), oops);
+        EXPECT_EQ(uv.value_or(3.14), 3);
+    }
```

Comment:
This looks identical to the other test you're adding.

Reply:
If you look very closely, the first one tests `E`, the second one tests `EV`. 🤓 (Same as their respective `makeUnexpected` siblings above each one.)

Reply:
Shouldn't review when I get up too early!